apps/falco/charts/falco-4.21.3/Chart.yaml

```diff
@@ -0,0 +1,9 @@
+apiVersion: v2
+name: falco
+description: A Helm chart to refer the official "Falco" helm chart
```

Comment:
Please change to "falco" as it needs to be consistent with chart name provided in st-charts.yaml config.

Reply:
@Josca 

Fixed!
I've also reduced the summary length, as suggested by the other CI jobs.

Please let me know if there are other issues that need to be addressed.

Thank you! :pray: